libs/elastic-explorer/src/lib/modules/search/components/filters-form/filters-form.component.ts

```diff
@@ -1,23 +1,28 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ViewChild } from '@angular/core';
 import { UntypedFormBuilder, UntypedFormControl, UntypedFormGroup } from '@angular/forms';
 import { ElasticAggregation, notNil, SelectOption, SelectOptionsProvider } from '@cognizone/model-utils';
 import { OnDestroy$ } from '@cognizone/ng-core';
 import { identity, Observable } from 'rxjs';
 import { filter, first, map, switchMap, toArray } from 'rxjs/operators';
 
 import { ElasticExplorerService } from '../../services/elastic-explorer.service';
+import { ElasticQueryEditorComponent } from "../elastic-query-editor/elastic-query-editor.component";
 
 @Component({
   selector: 'cz-filters-form',
   templateUrl: './filters-form.component.html',
   styleUrls: ['./filters-form.component.scss'],
 })
 export class FiltersFormComponent extends OnDestroy$ implements OnInit {
+  @ViewChild(ElasticQueryEditorComponent, { static: false }) // to be tested with true too
```

Comment:
So, did you test with `true`, does it work? Either way, that comment was for **you**, not for the codebase 😅

Reply:
forgot to remove the comment 😅 (doesn't work with true)